### PR TITLE
Issue/1712

### PR DESCRIPTION
**Pull request by micahnyc**
Added dwarf planet assets
Added dwarf planets to default_full
Changed color of ceres trail to match others.

### CODE REVIEW

File: data/assets/scene/solarsystem/dwarf_planets/haumea/label.asset [hunk 1]
```diff
@@ -0,0 +1,34 @@
+local Label = {
+  Identifier = "HaumeaLabel",
+  Renderable = {
+    Enabled = asset.Enabled,
+    Type = "RenderableLabel",
+    Text = "Haumea",
+    FontSize = 70.0,
+    Size = 9.10,
+    MinMaxSize = { 1, 100 },
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/haumea/label.asset:9]: -> `{ 1.0, 100.0 }`

File: data/assets/scene/solarsystem/dwarf_planets/haumea/planet.asset [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  Identifier = "dwarf_planet_models",
+  Version = 1
+})
+
+local Planet = {
+  Identifier = "HamueaModel",
+  Transform = {
+    Rotation = {
+      Type = "ConstantRotation",
+      RotationAxis = {0, 0, 1},
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/haumea/planet.asset:15]: -> `{ 0.0, 0.0, 1.0 }`

File: data/assets/scene/solarsystem/dwarf_planets/haumea/planet.asset [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      RotationRate = 1/(3.9154*60*60)
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/haumea/planet.asset:16]: -> `1.0`

File: data/assets/scene/solarsystem/dwarf_planets/haumea/trail.asset [hunk 4]
```diff
@@ -0,0 +1,28 @@
+local Trail = {
+  Identifier = "HaumeaTrail",
+  Renderable = {
+    Type = "RenderableTrailOrbit",
+    Color = { 0.2, 0.8, 0.3 },
+    Period = 90487.27692706819,
+    Resolution = 10000,
+    Enabled = asset.Enabled,
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/haumea/trail.asset:8]: -> `asset.enabled`

File: data/assets/scene/solarsystem/dwarf_planets/makemake/label.asset [hunk 5]
```diff
@@ -0,0 +1,34 @@
+local Label = {
+  Identifier = "MakemakeLabel",
+  Renderable = {
+    Enabled = asset.Enabled,
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/makemake/label.asset:4]: -> `asset.enabled`

File: data/assets/scene/solarsystem/dwarf_planets/makemake/label.asset [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    Type = "RenderableLabel",
+    Text = "Makemake",
+    FontSize = 70.0,
+    Size = 9.10,
+    MinMaxSize = { 1, 100 },
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/makemake/label.asset:9]: -> `{ 1.0, 100.0 }`

File: data/assets/scene/solarsystem/dwarf_planets/makemake/makemake.asset [hunk 7]
```diff
@@ -0,0 +1,61 @@
+local transforms = asset.require("scene/solarsystem/sun/transforms")
+local OneAU = 1.496e+8
+
+local trail = asset.require("./trail").Trail
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/makemake/makemake.asset:4]: Same as above

File: data/assets/scene/solarsystem/dwarf_planets/makemake/planet.asset [hunk 8]
```diff
@@ -0,0 +1,42 @@
+local sun = asset.require("scene/solarsystem/sun/sun")
+
+local modelFolder = asset.syncedResource({
+  Name = "Dwarf Planet Models",
+  Type = "HttpSynchronization",
+  Identifier = "dwarf_planet_models",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/makemake/planet.asset:6]: Same here;  `dwarf_planet_models_makemake` or `makemake_planet_models`

File: data/assets/scene/solarsystem/dwarf_planets/makemake/planet.asset [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+  Version = 1
+})
+
+local Planet = {
+  Identifier = "MakemakeGlobe",
+  Transform = {
+    Rotation = {
+      Type = "ConstantRotation",
+      RotationAxis = {0, 0, 1},
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/makemake/planet.asset:15]: -> `{ 0.0, 0.0, 1.0 }`

File: data/assets/scene/solarsystem/dwarf_planets/makemake/planet.asset [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+      RotationRate = 1/(10.273*60*60)
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/makemake/planet.asset:16]: -> `1.0`

File: data/assets/scene/solarsystem/dwarf_planets/orcus/label.asset [hunk 11]
```diff
@@ -0,0 +1,34 @@
+local Label = {
+  Identifier = "OrcusLabel",
+  Renderable = {
+    Enabled = asset.Enabled,
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/orcus/label.asset:4]: -> `asset.enabled`

File: data/assets/scene/solarsystem/dwarf_planets/orcus/label.asset [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+    Type = "RenderableLabel",
+    Text = "Orcus",
+    FontSize = 70.0,
+    Size = 9.10,
+    MinMaxSize = { 1, 100 },
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/orcus/label.asset:9]: -> `{ 1.0, 100.0 }`

File: data/assets/scene/solarsystem/dwarf_planets/orcus/orcus.asset [hunk 13]
```diff
@@ -0,0 +1,61 @@
+local transforms = asset.require("scene/solarsystem/sun/transforms")
+local OneAU = 1.496e+8
+
+local trail = asset.require("./trail").Trail
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/orcus/orcus.asset:4]: Same discussion as above

File: data/assets/scene/solarsystem/dwarf_planets/orcus/planet.asset [hunk 14]
```diff
@@ -0,0 +1,34 @@
+local sun = asset.require("scene/solarsystem/sun/sun")
+
+local Planet = {
+  Identifier = "OrcusGlobe",
+  Transform = {
+    Rotation = {
+      Type = "ConstantRotation",
+      RotationAxis = {0, 0, 1},
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/orcus/planet.asset:8]: -> `{ 0.0, 0.0, 1.0 }`

File: data/assets/scene/solarsystem/dwarf_planets/quaoar/planet.asset [hunk 15]
```diff
@@ -0,0 +1,34 @@
+local sun = asset.require("scene/solarsystem/sun/sun")
+
+local Planet = {
+  Identifier = "QuaoarGlobe",
+  Transform = {
+    Rotation = {
+      Type = "ConstantRotation",
+      RotationAxis = {0, 0, 1},
+      RotationRate = 1/(8.84*60*60)
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/quaoar/planet.asset:9]: -> `1.0`

File: data/assets/scene/solarsystem/dwarf_planets/quaoar/quaoar.asset [hunk 16]
```diff
@@ -0,0 +1,61 @@
+local transforms = asset.require("scene/solarsystem/sun/transforms")
+local OneAU = 1.496e+8
+
+local trail = asset.require("./trail").Trail
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/quaoar/quaoar.asset:4]: Same discussion as above

File: data/assets/scene/solarsystem/dwarf_planets/sedna/label.asset [hunk 17]
```diff
@@ -0,0 +1,34 @@
+local Label = {
+  Identifier = "SednaLabel",
+  Renderable = {
+    Enabled = asset.Enabled,
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/sedna/label.asset:4]: -> `asset.enabled`

File: data/assets/scene/solarsystem/dwarf_planets/sedna/label.asset [hunk 18]
```diff
@@ -0,0 +1,34 @@
...
+    Type = "RenderableLabel",
+    Text = "Sedna",
+    FontSize = 70.0,
+    Size = 9.10,
+    MinMaxSize = { 1, 100 },
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/sedna/label.asset:9]: -> `{ 1.0, 100.0 }`

File: data/assets/scene/solarsystem/dwarf_planets/sedna/planet.asset [hunk 19]
```diff
@@ -0,0 +1,34 @@
+local sun = asset.require("scene/solarsystem/sun/sun")
+
+local Planet = {
+  Identifier = "SednaGlobe",
+  Transform = {
+    Rotation = {
+      Type = "ConstantRotation",
+      RotationAxis = {0, 0, 1},
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/sedna/planet.asset:8]: -> `{ 0.0, 0.0, 1.0 }`

File: data/assets/scene/solarsystem/dwarf_planets/sedna/planet.asset [hunk 20]
```diff
@@ -0,0 +1,34 @@
...
+      RotationRate = 1/(10.273*60*60)
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/sedna/planet.asset:9]: -> `1.0`

File: data/assets/scene/solarsystem/dwarf_planets/sedna/planet.asset [hunk 21]
```diff
@@ -0,0 +1,34 @@
...
+    }
+  },
+  Renderable = {
+    Type = "RenderableGlobe",
+    Radii = { 995.0E3, 1060.0E3, 1025.0E3 },
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/sedna/planet.asset:14]: Easier to read: `{ 995000, 1060000, 1025000 }`

File: data/assets/scene/solarsystem/dwarf_planets/sedna/sedna.asset [hunk 22]
```diff
@@ -0,0 +1,61 @@
+local transforms = asset.require("scene/solarsystem/sun/transforms")
+local OneAU = 1.496e+8
+
+local trail = asset.require("./trail").Trail
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/sedna/sedna.asset:4]: Same discussion as above

File: data/assets/scene/solarsystem/dwarf_planets/sedna/trail.asset [hunk 23]
```diff
@@ -0,0 +1,28 @@
+local Trail = {
+  Identifier = "SednaTrail",
+  Renderable = {
+    Type = "RenderableTrailOrbit",
+    Color = { 0.2, 0.8, 0.3 },
+    Period = 4485811.755507208,
+    Resolution = 10000,
+    Enabled = asset.Enabled,
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/sedna/trail.asset:8]: -> `asset.enabled`

File: data/assets/scene/solarsystem/dwarf_planets/vesta/label.asset [hunk 24]
```diff
@@ -0,0 +1,34 @@
+local Label = {
+  Identifier = "VestaLabel",
+  Renderable = {
+    Enabled = asset.Enabled,
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/vesta/label.asset:4]: -> `asset.enabled`

File: data/assets/scene/solarsystem/dwarf_planets/vesta/label.asset [hunk 25]
```diff
@@ -0,0 +1,34 @@
...
+    Type = "RenderableLabel",
+    Text = "Vesta",
+    FontSize = 50.0,
+    Size = 8.5,
+    MinMaxSize = { 1, 100 },
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/vesta/label.asset:9]: -> `{ 1.0, 100.0 }`

File: data/assets/scene/solarsystem/dwarf_planets/vesta/planet.asset [hunk 26]
```diff
@@ -0,0 +1,44 @@
...
+  Identifier = "vesta_model",
+  Version = 1
+})
+
+local Planet = {
+  Identifier = "VestaModel",
+  Transform = {
+    Rotation = {
+      Type = "ConstantRotation",
+      RotationAxis = {0, 0, 1},
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/vesta/planet.asset:15]: -> `{ 0.0, 0.0, 1.0 }`

File: data/assets/scene/solarsystem/dwarf_planets/vesta/planet.asset [hunk 27]
```diff
@@ -0,0 +1,44 @@
...
+      RotationRate = 1/(5.34212766*60*60)
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/vesta/planet.asset:16]: -> `1.0`

File: data/assets/scene/solarsystem/dwarf_planets/vesta/planet.asset [hunk 28]
```diff
@@ -0,0 +1,44 @@
...
+    }
+  },
+  Renderable = {
+    Type = "RenderableModel",
+    GeometryFile = models .. "VestaComet_5000.obj",
+    SpecularIntensity = 0.0,
+    AmbientIntensity = 0.0,
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/vesta/planet.asset:23]: Why do these need special handling, but the other models do not?

File: data/assets/scene/solarsystem/dwarf_planets/vesta/trail.asset [hunk 29]
```diff
@@ -0,0 +1,28 @@
+local Trail = {
+  Identifier = "VestaTrail",
+  Renderable = {
+    Type = "RenderableTrailOrbit",
+    Color = { 0.2, 0.8, 0.3 },
+    Period = 1326.797192349944,
+    Resolution = 10000,
+    Enabled = asset.Enabled,
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/vesta/trail.asset:8]: -> `asset.enabled`

File: data/assets/scene/solarsystem/dwarf_planets/vesta/vesta.asset [hunk 30]
```diff
@@ -0,0 +1,61 @@
+local transforms = asset.require("scene/solarsystem/sun/transforms")
+local OneAU = 1.496e+8
+
+local trail = asset.require("./trail").Trail
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/vesta/vesta.asset:4]: Same discussion as for the other objects

File: data/assets/scene/solarsystem/dwarf_planets/vesta/vesta.asset [hunk 31]
```diff
@@ -0,0 +1,61 @@
...
+asset.onDeinitialize(function()
+  openspace.removeSceneGraphNode(trail)
+  openspace.removeSceneGraphNode(planet)
+  openspace.removeSceneGraphNode(label)
+  openspace.removeSceneGraphNode(position)
+end)
+
+
+asset.meta = {
+  Name = "Haumea (Keplerian)",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/dwarf_planets/vesta/vesta.asset:54]: Copy and paste error

**Review by alexanderbock — CHANGES_REQUESTED** [review]

**Comment by micahnyc** [comment]
[hamuea_model.zip](https://github.com/OpenSpace/OpenSpace/files/11195242/hamuea_model.zip)
[makemake_model.zip](https://github.com/OpenSpace/OpenSpace/files/11195244/makemake_model.zip)
[eris_model.zip](https://github.com/OpenSpace/OpenSpace/files/11195245/eris_model.zip)

**Comment by alexanderbock** [comment]
Seems odd that the labels for the dwarf planets are enabled where as the other planets are not in the default_full.  So we should disable those labels as well
image

**Pull request merged**